### DISCUSSION TITLE
Background color for exported maps

### DISCUSSION THREAD

**Discussion by grationd** [freeplane/freeplane#885] [Help] [answered]
Hello,

Nice to meet you all. I've been using Freeplane for a couple of days now and I noticed that exported maps with image or pdf formats won't show any background color that were applied in the editor. Is there any option to show the background color on exported maps? 

**Map in Freeplane:**

![Editor](https://user-images.githubusercontent.com/99073678/206877142-86e76a9b-81fa-4a58-87fa-5ec79e5e324f.png)

**A PNG export:**

![export](https://user-images.githubusercontent.com/99073678/206877000-a140d05a-a09d-4efd-875d-3e4c930b8412.png)

Thank you

**Comment by euu2021** [comment, marked as answer]
Disable `Preferences…->Appearance->Default colors->White background for printing`

**Reply by grationd** [reply]
Thank you
